### PR TITLE
IGNITE-4539: RocketMQ Streamer implementation.

### PR DESCRIPTION
**Pull request by shroman**

### CODE REVIEW

**Comment by vongosling** [comment]
@shroman  Have you design wiki for this feature ?

File: modules/rocketmq/pom.xml [hunk 1]
```diff
@@ -44,23 +44,14 @@
 
         <dependency>
             <groupId>org.apache.rocketmq</groupId>
-            <artifactId>rocketmq-client</artifactId>
+            <artifactId>rocketmq-broker</artifactId>
```
Comment (vongosling) [inline modules/rocketmq/pom.xml]: I have checked your broker dependency is just for test. Could you scope this dependency to test ?

Comment (shroman) [inline modules/rocketmq/pom.xml]: Thanks! Modified.

**Comment by shroman** [comment]
@vongosling It follows the standard streamer integration for Ignite. The docs on how to use will be available on https://apacheignite-mix.readme.io/docs.

**Comment by dspavlov** [comment]
IGNITE-4539 | Closed
-- | --

**Pull request closed without merging**